webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <ds-form ref="contributionForm" v-model="form" :schema="formSchema" @submit="submit">
+  <ds-form ref="contributionForm" v-model="form" :schema="formSchema">
     <template slot-scope="{ errors }">
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-07-10T22:05:59Z" title="Thursday, July 11th 2019, 12:05:59 am +02:00">Jul 11, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -10,13 +10,15 @@
           />
         </hc-teaser-image>
         <ds-input model="title" class="post-title" placeholder="Title" name="title" autofocus />
+        <small style="width:100%;position:relative;left:90%">{{ form.title.length }}/64</small>
```


I prefer to avoid inline styling @ogerly, not sure how others fell about it

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-18T09:04:59Z" title="Thursday, July 18th 2019, 11:04:59 am +02:00">Jul 18, 2019</time>_

---


Would be nice(!) to make a class out of it … @ogerly 
But not necessarily …
Inline styling has the positive side of better analysability, if the same style is not used elsewhere.
Classes (can) have descriptive names telling us what this is about.